miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt

```diff
@@ -0,0 +1,39 @@
+package com.rakuten.tech.mobile.miniapp.api
+
+import androidx.annotation.VisibleForTesting
+import com.rakuten.tech.mobile.miniapp.BuildConfig
+import com.rakuten.tech.mobile.sdkutils.RasSdkHeaders
+import com.rakuten.tech.mobile.sdkutils.okhttp.addHeaderInterceptor
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+internal fun createRetrofitClient(
+    baseUrl: String,
+    rasAppId: String,
+    subscriptionKey: String
+) = createRetrofitClient(
+    baseUrl = baseUrl,
+    headers = RasSdkHeaders(
+        appId = rasAppId,
+        subscriptionKey = subscriptionKey,
+        sdkName = "MiniApp",
+        sdkVersion = BuildConfig.VERSION_NAME
```

Comment:
Hmm, I don't think it's really necessary because this is the only place that these are being used currently. It would cause more misdirection in finding what these values are being set to.